salt/profiles/postgresql.sls

```diff
@@ -0,0 +1,39 @@
+profiles-db-user:
```

Comment:
I wonder if this can be extracted into `elife/postgresql-database.sls` to be used in addition to `elife/postgresql.sls`, by changing `pillar.profiles.db` to `pillar.elife.db`. It's the usual boilerplate we should stop repeating in every formula